spec/routes.spec.ts

```diff
@@ -69,7 +69,7 @@ const routeTests = ({
       const document = new OpenAPIGenerator(
         registry.definitions
       ).generateDocument(testDocConfig);
-      const responses = document[rootDocPath]['/'].get.responses;
+      const responses = document[rootDocPath]?.['/'].get.responses;
```

Comment:
Since "webhooks" is now optional and no longer any we can either bang it or optional coerce. Doesn't really make a difference to me as it's just a test.

Reply:
This is fine, at least we won't get "cannot read whatever of undefined" but as you said, it's a test